Code/Framework/AzCore/AzCore/std/string/conversions.h

```diff
@@ -286,6 +286,13 @@ namespace AZStd
         str = buf;
     }
 
+    template<class Str, class BoolType>
+    auto to_string(Str& str, BoolType value)
+        -> std::enable_if_t<AZStd::same_as<AZStd::remove_cvref_t<BoolType>, bool>>
+    {
+        str = value ? "true" : "false";
```

Comment:
Where is the return value here?

Reply:
std::enable_if_t should be `AZStd::enable_if_t`